homeassistant/components/climate/daikin.py

```diff
@@ -185,6 +185,7 @@ def set(self, settings):
         if values:
             self._force_refresh = True
             self._api.device.set(values)
+            self.async_update_ha_state(True)
```

Comment:
Don't use the async api in sync context. Since this is a polling entity a state update will be scheduled in the end of each service call, so this shouldn't be needed at all.

Reply:
Well, I need a away to tell ha that values have updated (there is no need to poll again)..

Reply:
You are right. I didn't need this, will update. Thank you for spotting it.